source/nodejs/adaptivecards/src/logging/log-enums.ts

```diff
@@ -0,0 +1,7 @@
+export enum LogLevel {
+	Warn,
+	Error,
+	Verbose,
```

Comment:
What is verbose?

Reply:
This buckets operational logs that a component might want to trace to provide additinoal debugging context - any log for eg that isn't an operational error or a warning event classifies as verbose.

Reply:
If any log that isn't Error or Warning classifies as Verbose, why are there also Info and Event? Are those nuances of Verbose? When is one supposed to use one over the other?